src/app/pages/task/task.page.html

```diff
@@ -13,23 +13,23 @@
     </ion-buttons>
   </ion-toolbar>
 </ion-header>
-
 <ion-content class="outer-content">
-  <div *ngIf="items">
+  <div *ngIf="items" class="item-container">
     <ion-list>
       <ion-item-group *ngFor="let item of items; let first = first; let last = last" [class.first-item]="first"
         [class.last-item]="last">
         <ion-item>
+          <ion-checkbox (click)="markItemStatus(item)" [checked]="item.status === 'COMPLETE'" class="ion-margin-end ion-align-items-start"></ion-checkbox>
```

Comment:
Common pattern will be to not put any business logic into the view and have method like `isChecked`